docs/input/docs/reference/build-servers/gitlab.md

```diff
@@ -13,10 +13,10 @@ A working example of integrating GitVersion with GitLab is maintained in the pro
 
 Here is a summary of what it demonstrated (many more details in the [README.md][])
 
-- Is a working example know as a [Guided Explorations (GE Manifesto)][] - so job logs and package artifacts can be reviewed. The project can also be imported to your own GitLab group or instance.
+- Is a reusable working example known as a Guided Exploration [(GE Manifesto)][] - so job logs and package artifacts can be reviewed. The project can also be imported to your own GitLab group or instance as a starting point for your own work.
 - IMPORTANT: It demonstrates how to override GitLab CI's default cloning behavior so that GitVersion can do a dynamic copy. Selectively clones GitVersion.yml so that these settings take effect. This best practice demonstrates the best way to do this while avoiding a double-cloning of the project (once by GitLab Runner and once by GitVersion). 
```

Comment:
```suggestion
- _Important_: It demonstrates how to override GitLab CI's default cloning behavior so that GitVersion can do a dynamic copy. Selectively clones GitVersion.yml so that these settings take effect. This best practice demonstrates the best way to do this while avoiding a double-cloning of the project (once by GitLab Runner and once by GitVersion). 
```